test/drivers/test_suites/syscall_exit_suite/io_uring_setup_x.cpp

```diff
@@ -0,0 +1,141 @@
+#include "../../event_class/event_class.h"
+
+#if defined(__NR_io_uring_setup)
+
+#include <linux/io_uring.h>
+
+TEST(SyscallExit, io_uring_setupX)
+{
+	auto evt_test = get_syscall_event_test(__NR_io_uring_setup, EXIT_EVENT);
+
+	evt_test->enable_capture();
+
+	/*=============================== TRIGGER SYSCALL ===========================*/
+
+    /* There could be cases in which the structure `io_uring_params`
+     * doesn't have the filed `feature`, so define it to `0` as default.
+     */
+    uint32_t expected_features = 0;
```

Comment:
minor/typo: could you check the indent/formatting of `expected_features` here and below?

Reply:
done :)